diff --git a/ZLIB/Makefile b/ZLIB/Makefile
index 59b1b55..bae6942 100644
--- a/ZLIB/Makefile
+++ b/ZLIB/Makefile
@@ -1,5 +1,5 @@
 ZLIB_TARGET=libmyz.a
-ZLIB_SOURCE=adler32.c compress.c crc32.c deflate.c gzio.c infblock.c infcodes.c inffast.c inflate.c inftrees.c infutil.c trees.c uncompr.c zutil.c
+ZLIB_SOURCE=adler32.c compress.c crc32.c deflate.c gzclose.c gzlib.c gzread.c gzwrite.c infback.c inffast.c inflate.c inftrees.c trees.c uncompr.c zutil.c
 
 COMPILER ?= gcc
 #COMPILER ?= clang
diff --git a/ZLIB/gzguts.h b/ZLIB/gzguts.h
index eba7208..6b60b4f 100644
--- a/ZLIB/gzguts.h
+++ b/ZLIB/gzguts.h
@@ -212,3 +212,5 @@ char ZLIB_INTERNAL *gz_strwinerror(DWORD error);
    (possible z_off64_t types off_t, off64_t, and long are all signed) */
 unsigned ZLIB_INTERNAL gz_intmax(void);
 #define GT_OFF(x) (sizeof(int) == sizeof(z_off64_t) && (x) > gz_intmax())
+
+#include <unistd.h>
